Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11-slim-buster
+
+LABEL Name='Python Template Python'
+LABEL Version=0.1
+LABEL at.strg.docker.agents.maintainer=""
```

Comment:
consider removing these custom labels as they create no benefit and require additional maintenance

Reply:
please check my comment on precommit file, this is why it breaks, the dockerlint is not usefull

Reply:
custom labels are removed